<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrescriptionsFrame</class>
 <widget class="QFrame" name="PrescriptionsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="medicationBox">
     <property name="title">
      <string>Prescriptions search</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="medicationNameLabel">
        <property name="text">
         <string>MEDICATION</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="medicationNameField">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lotLabel">
        <property name="text">
         <string>LOT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lotField">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="filledLabel">
        <property name="text">
         <string>FILLED</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="filledField">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>1970</year>
          <month>1</month>
          <day>31</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="patientBox">
     <property name="title">
      <string>Patient search</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lastLabel">
          <property name="text">
           <string>LAST NAME</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lastField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="firstLabel">
          <property name="text">
           <string>FIRST NAME</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firstField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="dobLabel">
          <property name="text">
           <string>DOB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dobField">
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1970</year>
            <month>1</month>
            <day>31</day>
           </datetime>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="activeField">
          <property name="text">
           <string>Active</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTableWidget" name="resultTable">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>11</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>MR NUMBER</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LAST NAME</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FIRST NAME</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DOB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>MEDICATION</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FORM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STRENGTH</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>AMOUNT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>WRITTEN</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FILLED</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>LOT</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="modifyAction">
   <property name="text">
    <string>Modify Prescription</string>
   </property>
   <property name="toolTip">
    <string>Modify past prescription information</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>Print Label</string>
   </property>
   <property name="toolTip">
    <string>Print a label for the prescription</string>
   </property>
  </action>
  <action name="invalidateAction">
   <property name="text">
    <string>Invalidate</string>
   </property>
   <property name="toolTip">
    <string>Marks the prescription as invalid.</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>medicationNameField</tabstop>
  <tabstop>lotField</tabstop>
  <tabstop>filledField</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>resultTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>medicationNameField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>lotField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>lastField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>firstField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>medicationNameField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lotField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>firstField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
